_field-types/supported-field-types/knn-vector.md

```diff
@@ -22,19 +22,19 @@ PUT test-index
 {
   "settings": {
     "index": {
-      "knn": true,
-      "knn.algo_param.ef_search": 100
+      "knn": true
     }
   },
   "mappings": {
     "properties": {
       "my_vector": {
         "type": "knn_vector",
         "dimension": 3,
+        "space_type": "l2",
```

Comment:
I think on this example we should give a best default experience. Which is no mode, no compression, just spaceType, dim and type attributes. What you think?

Reply:
sure - only thing is that I believe defaults will be picked up from index_settings in this case.